source/agora/network/NetworkManager.d

```diff
@@ -524,10 +524,20 @@ public class NetworkManager
     public bool getNetTimeOffset (uint threshold, out long time_offset)
         @safe nothrow
     {
-        static time_t[] offsets;
+        static struct TimeInfo
```

Comment:
For log output? How about adding a brief document about TimeInfo?

Reply:
Ok I added a comment.